Make.config

```diff
@@ -35,7 +35,7 @@ $(TOP)/Make.config.inc: $(TOP)/Make.config $(TOP)/mk/mono.mk
 
 include $(TOP)/Make.versions
 
-APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/d16-6/b3eedfed9b6c05bf8642cebe9704159ce340d600/45/package/bundle.zip
+APIDIFF_REFERENCES=https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/d16-6/ce0cc74a3bae25dbe29762f500a38b0325d2ea70/7/package/bundle.zip
```

Comment:
@whitneyschmidt you did not replace the full URL so the branch name is incorrect

https://bosstoragemirror.blob.core.windows.net/wrench/jenkins/d16-6-xcode11.6/ce0cc74a3bae25dbe29762f500a38b0325d2ea70/7/package/bundle.zip

Reply:
❗Good catch! Fixed.